tasks/vm_state_present.yml

```diff
@@ -49,7 +50,7 @@
     bootable: "{{ item.1.bootable | default(omit) }}"
     activate: "{{ item.1.activate | default(omit) }}"
     wait: true
-  # When there is no profile with disks it will be used
+  # When there is profile with disks, vms disks will update them
   when: item.0.profile is defined and item.0.profile.disks is defined
```

Comment:
This conditional doesn't make sense to me. Why I can't define profile without additional disk and add them in vms definitions? 

I think that other missing paramters from templates can be added in vms definitions, so I would like to have this option at defining disk also.